docs/sphinx/conf.py

```diff
@@ -36,6 +36,7 @@
 ]
 
 pygments_style = "sphinx"
+pygments_dark_style = "monokai"
```

Comment:
This is required to make sure code blocks such as the one in HttpHeaders are visible in dark mode.